oscarlab/graphene
# [Jenkins] Test 'make clean' also on 18.04

## Affected components


## Description of the changes 

See commit messages.

## How to test this PR? 

See Jenkins.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1290)
